packages/world-modules/src/interfaces/IBaseWorld.sol

```diff
@@ -6,11 +6,13 @@ pragma solidity >=0.8.24;
 import { IStore } from "@latticexyz/store/src/IStore.sol";
 import { IWorldKernel } from "@latticexyz/world/src/IWorldKernel.sol";
 
+import { IUnstable_DelegationWithSignatureSystem } from "./IUnstable_DelegationWithSignatureSystem.sol";
+
 /**
  * @title IBaseWorld
  * @author MUD (https://mud.dev) by Lattice (https://lattice.xyz)
  * @notice This interface integrates all systems and associated function selectors
  * that are dynamically registered in the World during deployment.
  * @dev This is an autogenerated file; do not edit manually.
  */
-interface IBaseWorld is IStore, IWorldKernel {}
+interface IBaseWorld is IStore, IWorldKernel, IUnstable_DelegationWithSignatureSystem {}
```

Comment:
need to add this system to the excludes of the mud.config

Reply:
done!